evm/v0.5/contracts/dev/Whitelisted.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity 0.5.0;
+
+import "./Owned.sol";
+
+/**
+ * @title Whitelisted
+ * @notice Allows the owner to add and remove addresses from a whitelist
+ */
+contract Whitelisted is Owned {
+
+  mapping(address => bool) public whitelisted;
+
+  event AddedToWhitelist(address user);
+  event RemovedFromWhitelist(address user);
+
+  /**
+   * @notice Adds an address to the whitelist
+   * @param _user The address to whitelist
+   */
+  function addToWhitelist(address _user) external onlyOwner() {
+    whitelisted[_user] = true;
+    emit AddedToWhitelist(_user);
+  }
+
+  /**
+   * @notice Removes an address from the whitelist
+   * @param _user The address to remove
+   */
+  function removeFromWhitelist(address _user) external onlyOwner() {
```

Comment:
Part of me wants to condense `add` and `remove` into one method that accepts a boolean, because it won't clutter up the inheriting contracts. It might be a little more error prone though, so I'm leaning toward leaving it as is.